portfolio/src/main/webapp/discussionScript.js

```diff
@@ -13,16 +13,44 @@ function displayList() {
       content.classList.add("post-content");
       content.textContent = data[i].content;
 
-
       let singlePost = document.createElement('div');
       singlePost.classList.add("post");
 
+      let replyButton = document.createElement('button');
+      replyButton.textContent = "Reply to this post";
+
       singlePost.appendChild(title);
       singlePost.appendChild(content);
+      singlePost.appendChild(replyButton);
 
       posts.appendChild(singlePost);
+
+      replyButton.addEventListener("click", () => {
+        replyButton.classList.add("hidden");
+        createReplyForm(singlePost, data[i].id);
+      });
     }
 
     document.getElementById('discussion-container').appendChild(posts);
   })
 }
+
+function createReplyForm(singlePost, postId){
+
+  let replyForm = document.createElement("form");
+  replyForm.action = `/reply?postID=${postId}`;
```

Comment:
I don't see this route being defined anywhere, does this do anything yet?

Reply:
Not yet, that's what I'm working on right now. Just wanted to straighten out the front-end first.

Reply:
Gotcha, lgtm!